x-pack/metricbeat/module/azure/_meta/kibana/7/dashboard/Metricbeat-azure-database-account-overview.json

```diff
@@ -0,0 +1,715 @@
+              "query": ""
+            }
+          }
+        },
+        "title": "Database Account Index Usage [Metricbeat Azure]",
+        "uiStateJSON": {},
+        "version": 1,
+        "visState": {
+          "aggs": [],
+          "params": {
+            "axis_formatter": "number",
+            "axis_min": 0,
+            "axis_position": "left",
+            "axis_scale": "normal",
+            "default_index_pattern": "metricbeat-*",
+            "default_timefield": "@timestamp",
+            "filter": {
+              "language": "kuery",
+              "query": "azure.resource.type : \"Microsoft.DocumentDb/databaseAccounts\"   "
+            },
+            "id": "e9a40230-32e9-11ea-bda2-69435df36a5c",
+            "index_pattern": "metricbeat-*",
+            "interval": "\u003e=5m",
+            "isModelInvalid": false,
+            "series": [
+              {
+                "axis_position": "right",
+                "chart_type": "line",
+                "color": "#3185FC",
+                "fill": "0",
+                "filter": {
+                  "language": "kuery",
+                  "query": ""
+                },
+                "formatter": "bytes",
+                "id": "e9a40231-32e9-11ea-bda2-69435df36a5c",
+                "label": "Data Usage",
+                "line_width": "2",
+                "metrics": [
+                  {
+                    "field": "azure.database_account.index_usage.total",
+                    "id": "e9a40232-32e9-11ea-bda2-69435df36a5c",
+                    "type": "avg"
+                  }
+                ],
+                "point_size": 0,
+                "separate_axis": 0,
+                "split_color_mode": "rainbow",
+                "split_mode": "terms",
+                "stacked": "none",
+                "terms_exclude": "\"\u003cempty\u003e\"",
+                "terms_field": "azure.dimensions.database_name",
+                "terms_order_by": "_count",
+                "type": "timeseries",
+                "value_template": "{{value}} ms"
+              }
+            ],
+            "show_grid": 1,
+            "show_legend": 1,
+            "time_field": "@timestamp",
+            "type": "timeseries"
+          },
+          "title": "Database Account Index Usage [Metricbeat Azure]",
+          "type": "metrics"
+        }
+      },
+      "id": "d2801d70-8490-11ea-b181-4b1a9e0110f9",
+      "migrationVersion": {
+        "visualization": "7.4.2"
+      },
+      "references": [],
+      "type": "visualization",
+      "updated_at": "2020-04-22T12:00:12.999Z",
+      "version": "WzcwNjksMTJd"
+    },
+    {
+      "attributes": {
+        "description": "",
+        "kibanaSavedObjectMeta": {
+          "searchSourceJSON": {}
+        },
+        "title": "Database Account Filters [Metricbeat Azure]",
+        "uiStateJSON": {},
+        "version": 1,
+        "visState": {
+          "aggs": [],
+          "params": {
+            "controls": [
```

Comment:
Perhaps it makes sense to extract `DatabaseName` dimension as a dedicated input. This will allow to focus the dashboard on a specific database.

Reply:
added the database as a filter